### PR TITLE
improve dependent keys ux

### PR DESCRIPTION
**Pull request by patricklx**
## Description
this now also marks the changed dependent keys with a red text colour and adds a tooltip

## Screenshots
Screenshot 2024-02-10 at 00 22 30

### CODE REVIEW

File: app/components/object-inspector/dependent-keys.hbs [hunk 1]
```diff
@@ -1,14 +1,17 @@
 <li class="mixin__property-dependency-list relative">
   {{svg-jar "dependent-key-connection" width="20px" height="10px"}}
   <ul class="m-0 p-0 list-none">
-    {{#each @keys as |depKey|}}
-      <li class="mixin__property-dependency-item relative text-base12 text-sm">
-      {{#if (match depKey "•")}}
+    {{#each @keys as |dep|}}
+      <li
+              class="mixin__property-dependency-item relative text-base12 text-sm {{if dep.changed 'text-spec01'}}"
```
Comment (RobbieTheWagner) [inline app/components/object-inspector/dependent-keys.hbs:6]: It looks like there is extra spacing here, can we remove that?

**Comment by RobbieTheWagner** [comment]
Would it be possible to get a screenshot please?

File: app/components/object-inspector/dependent-keys.hbs (hunk 1 above)
Comment (patricklx) [reply app/components/object-inspector/dependent-keys.hbs]: fixed

**Comment by patricklx** [comment]
added screemshot

**Comment by RobbieTheWagner** [comment]
Can you please explain what the idea is here? I don't find it very intuitive from the screenshot what it is that we are gaining.

**Comment by patricklx** [comment]
this is a follow up https://github.com/emberjs/ember-inspector/pull/2546

**Comment by RobbieTheWagner** [comment]
Are we able to make the title show up closer to the property that changed? It seems unclear what it belongs to in the screenshot.

**Comment by patricklx** [comment]
That's the native tooltip. I can try to make another screen shot with the cursor at the right position. Need to find the right keycombo...

**Comment by RobbieTheWagner** [comment]
> That's the native tooltip. I can try to make another screen shot with the cursor at the right position. Need to find the right keycombo...

If it matches up to the correct item in reality, then that is fine with me.

**Review by RobbieTheWagner — APPROVED** [review]

**Pull request merged**